client/verta/verta/_registry/modelversions.py

```diff
@@ -45,10 +48,15 @@ def _create_element(self, msg):
 
     def with_model(self, registered_model=None):
         new_list = copy.deepcopy(self)
+        new_list._msg = _RegistryService.FindModelVersionRequest()
         if registered_model:
             new_list._msg.id.registered_model_id = registered_model.id
         else:
             new_list._msg.id.registered_model_id = 0
+        new_list._msg.predicates.extend(self._msg.predicates)
+        new_list._msg.pagination.CopyFrom(self._msg.pagination)
+        new_list._msg.ascending = self._msg.ascending
+        new_list._msg.sort_key = self._msg.sort_key
```

Comment:
So @convoliution these lines is needed because named_id should be removed from a request to use model id: https://vertaai.slack.com/archives/CS7UJ399B/p1599144371104200?thread_ts=1599134352.102400&cid=CS7UJ399B

Reply:
It works. Thank you

Reply:
@convoliution